lib/core/engine/command/measure.js

```diff
@@ -71,14 +71,22 @@ class Measure {
       await setOrangeBackground(this.browser.getDriver(), this.options);
     } else {
       await setOrangeBackground(this.browser.getDriver(), this.options);
+
+      // Give ffmpeg/video on phone time to settle
+      if (isAndroidConfigured(this.options)) {
+        await delay(ANDROID_DELAY_TIME);
+      } else {
+        await delay(400);
+      }
+
       await this.video.record(numberOfMeasuredPages, index);
     }
 
     // Give ffmpeg/video on phone time to settle
     if (isAndroidConfigured(this.options)) {
-      return delay(ANDROID_DELAY_TIME);
+      await delay(ANDROID_DELAY_TIME);
```

Comment:
This looks fine, but seems unnecessary since we're already waiting on _startVideo?